src/ipsis/course-merge/CourseMergeOfferingCollectionEntity.js

```diff
@@ -60,5 +60,17 @@ export class CourseMergeOfferingCollectionEntity extends Entity {
 
 		return this._entity.getLinkByRel('prev').href;
 	}
+
+	filtersHref() {
+		if (!this._entity.hasLinkByRel(Rels.filters)) {
```

Comment:
Minor nit, should we add a helper method like `hasNextPage` and `hasPrevPage` (e.g.., `hasFilters`)?